backend/src/main/java/com/woowacourse/kkogkkog/coupon/presentation/CouponController.java

```diff
@@ -77,6 +77,13 @@ public ResponseEntity<CouponsResponse> save(@LoginMemberId Long loginMemberId,
         return ResponseEntity.created(null).body(new CouponsResponse(responses));
     }
 
+    @PostMapping("/code")
+    public ResponseEntity<CouponResponse> saveByCouponCode(@LoginMemberId Long loginMemberId,
+                                                           @RequestBody String couponCode) {
+        CouponResponse couponResponse = couponService.saveByCouponCode(loginMemberId, couponCode);
```

Comment:
개인적으로 `UnregisteredCoupon`이라는 도메인명에 맞게 register라는 명칭을 더 활용하면 좋을 듯합니다.

```suggestion
    public ResponseEntity<CouponResponse> registerCouponCode(@LoginMemberId Long loginMemberId,
                                                           @RequestBody String couponCode) {
        CouponResponse couponResponse = couponService.saveByCouponCode(loginMemberId, couponCode);
```